assets/data-port/import/upload/component.js

```diff
@@ -0,0 +1,51 @@
+import { H, Section } from '@woocommerce/components';
+import { __ } from '@wordpress/i18n';
+import UploadLevels from '../upload-level';
+import { Notice } from '../../notice';
+import { Button } from '@wordpress/components';
+
+/**
+ * This component displays the upload page of the importer.
+ */
+export const UploadPage = ( { state, isReady, submitStartImport } ) => {
```

Comment:
The same question about the file name here.

Reply:
Changed it in the same commit 1333b18d329cc2be12b3e0ead86ac544d1678781